src/components/Loader/Loader.tsx

```diff
@@ -8,19 +8,10 @@ const Loader = () => (
 
     <div className={styles.container}>
       <div className={styles.planet}>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
-        <div className={styles.circle}></div>
+        <img src="/assets/images/earth.svg" alt=" " />
```

Comment:
Question: should the alt tag here be something like `alt="earth"` or `alt="cartoon representation of earth from space"`? Does that type of thing matter for a loader?

Reply:
the loader itself has a screenreader label that serves in place of the whole thing

Reply:
I made it empty on purpose because we specifcally *don't* want a screenreader to output something like this

> loading.... Image: earth image: sattelite

we'd just want it to say `loading...` to them